proposals/devcontainer-features-distribution.md

```diff
@@ -136,15 +136,24 @@ The `.tgz` archive file must be named `devcontainer-feature-<featureId>.tgz`.
 
 ### Locally referenced Features
 
-To aid in feature authorship, or in instances where a feature should not be published externally, individual features can be referenced locally from the project's file tree.
+Instead of publishing a feature to an OCI registry, a feature's source code may be referenced from a local folder.  Locally referencing a feature may be useful when first authoring a feature.
```

Comment:
```suggestion
Instead of publishing a feature to an OCI registry, a feature's source code may be referenced from a local folder. Locally referencing a feature may be useful when first authoring a feature.
```

Super nit 😄

Reply:
Also nit: capitalizing all instances of "Feature" 😄